core/src/main/java/com/novoda/merlin/MerlinBuilder.java

```diff
@@ -42,8 +39,8 @@
      * @return MerlinBuilder.
      */
     public MerlinBuilder withConnectableCallbacks() {
-        connectableRegisterer = new MerlinRegisterer<>();
-        this.merlinConnector = new Connector(connectableRegisterer);
+        connectables = new Register<>();
+        this.merlinConnector = new ConnectCallbackManager(connectables);
```

Comment:
These two lines, at least for me, making instantly more sense than they did before.